core/src/main/java/io/questdb/griffin/model/QueryModel.java

```diff
@@ -1320,6 +1333,10 @@ public void setWhereClause(ExpressionNode whereClause) {
         this.whereClause = whereClause;
     }
 
+    public void setBetweenType(int betweenType) {
+        this.betweenType = betweenType;
+    }
+
     @Override
     public void toSink(@NotNull CharSink<?> sink) {
```

Comment:
I didn't quite get this one.

Reply:
Hey, while testing I am getting this error repeatedly. 
``The type jdk.internal.math.FDBigInteger is not accessible``
and other errors related to ``jdk.internal``.

Reply:
![image](https://github.com/user-attachments/assets/4b463fe8-7a98-4d51-bf1d-1f7984d2c956)
It throws in  ``/questdb/core/src/main/java/io/questdb/std/Numbers.java``

Reply:
I used [this](https://github.com/questdb/questdb/pull/4733) and tried to write tests and update the parser but I can't test because of these errors. Should I build the project again, but I'll lose all the changes? How do I proceed? Do you have any suggestions

Reply:
Hey, I did reclone it, but it still shows the same errors, I think I need to clone it from the main repo, since there are only 60-70 lines of changes I can redo them after cloning the main repo. But I am not sure about this pr, do I need to create a new one and delete this one? Also, I am sorry for not doing anything in the past week as I was busy with my school exams. Thank you!

Reply:
I will close this pr and create a new one!!!